define/field_type.date.js

```diff
@@ -1,2 +1,26 @@
-module.exports = function(field_type_date){
-}
\ No newline at end of file
+var Promise = require("bluebird");
+
+module.exports = function(field_type_date) {
+
+    field_type_date.prototype.isProperValue = function(date) {
+        return new Promise(function(resolve, reject) {
+
+            var date_in_string = date.toString();
+
+            var regex = /^[1-2][0-9]{3}-([1-9]|0[1-9]|1[0-2])-([1-9]|0[1-9]|1[0-9]|2[0-9]|30|31)$/; //granulation_per_day
```

Comment:
Ok kwestia poprawienia regex'a dla date, bo akceptuje tylko pełne daty YYYY-MM-DD.

Jeśli chodzi o 0-999 n.e to zapisujemy dopełniając zerami z lewej strony. Np. rok 20 n.e  to 0020.
p.n.e określamy z -.
Szczerze mówiąc nie zajmowałbym się jakimś archaicznymi datami. Maksymalny zakres jaki obsługuje 
Date.parse to 1 stycznia 100r. n.e.

```javascript
Date.parse("100-1-1")
-59011462800000
var d = new Date(-59011462800000)
undefined
d
Fri Jan 01 100 00:00:00 GMT+0100 (CET)
```